pkg/arvo/lib/bip32.hoon

```diff
@@ -212,5 +212,5 @@
 ::
 ++  hash160
   |=  d=@
-  (ripemd-160:ripemd:crypto 256 (sha-256:sha d))
+  (ripemd-160:ripemd:crypto 32 (sha-256:sha d))
```

Comment:
found another bug!

I don't need to build a new pill for this, because it's not in `sys`, right?

Reply:
Yup, that's right.